src/main/java/org/broadinstitute/hellbender/engine/dataflow/coders/UUIDCoder.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.hellbender.engine.dataflow.coders;
+
+import com.google.cloud.dataflow.sdk.coders.DelegateCoder;
+import com.google.cloud.dataflow.sdk.coders.StringUtf8Coder;
+
+import java.util.UUID;
+
+/**
+ * This is basically StringDelegateCoder, except that we need to call fromString instead of a one arg constructor.
+ */
+public class UUIDCoder {
```

Comment:
i'm not happy about having a class with exactly 1 constant field in it. This field should do somewhere else and the class should go away i think.